src/datasources/accounts/accounts.datasource.ts

```diff
@@ -52,17 +66,21 @@ export class AccountsDatasource implements IAccountsDatasource, OnModuleInit {
     );
   }
 
-  async createAccount(address: `0x${string}`): Promise<Account> {
+  async createAccount(args: {
+    address: `0x${string}`;
+    clientIp: string;
```

Comment:
As this function's arity increased to 2, I've wrapped the arguments on a type, which caused some additional changes in the PR.